app/src/models/popup.ts

```diff
@@ -121,3 +124,12 @@ export type Popup =
       pullRequest: PullRequest
     }
   | { type: PopupType.MergeConflicts; repository: Repository }
+  | {
+      type: PopupType.OversizedFiles
+      dispatcher: Dispatcher
+      fileList: string[]
```

Comment:
Could this be a more descriptive field than `fileList`? Maybe `largeFiles` or `filesExceedingLimit`?